arch/arm/soc/nordic_nrf/nrf52/soc.c

```diff
@@ -381,7 +381,7 @@ static int nordicsemi_nrf52_init(struct device *arg)
 
 	key = irq_lock();
 
-#ifdef CONFIG_SOC_NRF52832
+#if defined(CONFIG_SOC_NRF52832) || defined(CONFIG_SOC_NRF52810)
 	nordicsemi_nrf52832_init();
```

Comment:
I think that both chips shall have different init functions. Most of PANs were fixed in nrf52810

Reply:
correct.

Reply:
Fixed